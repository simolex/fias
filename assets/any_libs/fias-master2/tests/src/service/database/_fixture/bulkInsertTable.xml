<?xml version="1.0" ?>
<dataset>
    <table name="bulkInsertTable">
        <column>id</column>
        <column>row1</column>
        <column>row2</column>
    </table>
</dataset>
