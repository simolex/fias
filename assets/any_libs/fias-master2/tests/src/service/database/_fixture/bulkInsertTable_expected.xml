<?xml version="1.0" ?>
<dataset>
    <table name="bulkInsertTable">
        <column>id</column>
        <column>row1</column>
        <column>row2</column>
        <row>
            <value>1</value>
            <value>row 1 1</value>
            <value>row 1 2</value>
        </row>
        <row>
            <value>2</value>
            <value>row 2 1</value>
            <value>row 2 2</value>
        </row>
        <row>
            <value>3</value>
            <value>row 3 1</value>
            <value>row 3 2</value>
        </row>
        <row>
            <value>4</value>
            <value>row 4 1</value>
            <value>row 4 2</value>
        </row>
        <row>
            <value>5</value>
            <value>row 5 1</value>
            <value>row 5 2</value>
        </row>
    </table>
</dataset>
